stdlib/3/csv.pyi

```diff
@@ -1,73 +1,74 @@
-from typing import Any, Iterable
+from collections import OrderedDict
+import sys
+from typing import Any, Dict, Iterable, Iterator, List, Optional, Sequence
 
-QUOTE_ALL = ...  # type: int
-QUOTE_MINIMAL = ...  # type: int
-QUOTE_NONE = ...  # type: int
-QUOTE_NONNUMERIC = ...  # type: int
+from _csv import _reader, _writer, reader, writer, register_dialect, unregister_dialect, get_dialect, list_dialects, field_size_limit, QUOTE_ALL, QUOTE_MINIMAL, QUOTE_NONE, QUOTE_NONNUMERIC
```

Comment:
To re-export these here, you need to write `from _csv import reader as reader` and similar.